### ISSUE TITLE
Function with Sequential return value cannot define as function schema

### ISSUE THREAD

**Issue by MokkeMeguru** [metosin/malli#585]
Problem Overview

```clojure
(defn gen-one [num-times]
  (repeat num-times 1))
(m/validate [:* [:= 1]] (gen-one 10)) ;; true

(m/=> gen-one [:=> [:cat int?] [:* [:= 1]]]) 
;; will cause
;; Execution error (ExceptionInfo) at malli.core/-fail! (core.cljc:119).
;; :malli.core/invalid-schema {:schema nil}
```

and Also

```clojure
(defn wrapped-gen-one [num-times]
  {:result (repeat num-times 1)})

(m/=> wrapped-gen-one [:=> [:cat int?] [:map [:result [:* [:= 1]]]]])
;; will cause
;; Execution error (ExceptionInfo) at malli.core/-fail! (core.cljc:119).
;; :malli.core/invalid-schema {:schema nil}
```

Question 

How to define the function spec with sequential return value?

**Comment by MokkeMeguru** [comment]
I found when this problem caused, I enabled malli.clj-kondo, I will throw the smallest project soon.

**Comment by ikitommi** [comment]
fixed in `[metosin/malli "0.7.2"]`
